src/_xoConnection.js

```diff
@@ -131,6 +131,35 @@ class XoConnection extends Xo {
     return id;
   }
 
+  async runBackupJob(jobId, scheduleId, { remotes, nExecutions = 1 }) {
+    for (let i = 0; i < nExecutions; i++) {
+      await xo.call("backupNg.runJob", { id: jobId, schedule: scheduleId });
+    }
+    const backups = {};
+    if (remotes) {
+      const backupsByRemote = await xo.call("backupNg.listVmBackups", {
+        remotes,
+      });
+      for (const remote in backupsByRemote) {
+        backups[remotes] = [];
```

Comment:
You can store the reference in a `const`

Reply:
Even if it is used once?

Reply:
It just for not access multiple times to the property. It's not a big deal you can ignore this commentary.